### PR TITLE
Cleanup ameba warnings

### PR DESCRIPTION
**Pull request by Sija**
Followup to #520, #538

### CODE REVIEW

File: src/kemal/ext/response.cr [hunk 1]
```diff
@@ -1,6 +1,7 @@
 class HTTP::Server::Response
   class Output
     def close
+      # ameba:disable Style/NegatedConditionsInUnless
       unless response.wrote_headers? && !response.headers.has_key?("Content-Range")
```
[the remark this answers is not in the document]
Comment (Sija) [reply src/kemal/ext/response.cr]: See #538

File: src/kemal/static_file_handler.cr [hunk 2]
```diff
@@ -4,6 +4,7 @@
 
 module Kemal
   class StaticFileHandler < HTTP::StaticFileHandler
+    # ameba:disable Metrics/CyclomaticComplexity
```
[the remark this answers is not in the document]
Comment (Sija) [reply src/kemal/static_file_handler.cr]: I'd leave it for another PR.

**Comment by Sija** [comment]
@sdogruyol 🏓

File: src/kemal/ext/response.cr (hunk 1 above)
[the remark this answers is not in the document]
Comment (sdogruyol) [reply src/kemal/ext/response.cr]: Please let's not do that again 😸

[the remark this answers is not in the document]
Comment (veelenga) [reply src/kemal/ext/response.cr]: The correct way of using `if` is:

`if !response.wrote_headers? || !response.headers.has_key?("Content-Range")`

**Pull request merged**